src/Swift.Runtime/src/Swift/AnyType.cs

```diff
@@ -17,28 +18,29 @@ static TypeMetadata ISwiftObject.GetTypeMetadata()
     {
         return TypeMetadata.Zero;
     }
-    public AnyType(SwiftHandle payload)
+    public AnyType(IntPtr payload)
```

Comment:
This could potentially conflict with a constructor taking a single machine width integer

Reply:
Resolved by passing a wrapper type.